README.md

```diff
@@ -67,6 +67,25 @@ The following attributes are available for all models of `viamrtsp` cameras:
 
 To test your camera, go to the [**CONTROL** tab](https://docs.viam.com/fleet/control/) of your machine in the [Viam app](https://app.viam.com) and expand the camera's panel.
 
+## RTSP stream discovery
+### Use `DiscoverComponents`
+In addition to being able to manually specify RTSP addresses to stream from, `viamrtsp` also offers a discovery utility to search for IP cameras connected to your LAN and return their respective RTSP addresses.
+
+Each model in `viamrtsp` registers a `Discover` method that can be invoked via a `DiscoverComponents` robot API call. See [examples/discovery-client](examples/discovery-client/client.go) for an example usage of discovery and [`DiscoverComponents`](https://docs.viam.com/appendix/apis/robot/#discovercomponents) for documentation.
+
+As of time of writing, Viam supports `DiscoverComponents` in Flutter and Golang.
+
+### Common RTSP discovery pitfalls
+#### DHCP
+IP camera does not support DHCP, and does not have an assigned IP after connecting to your LAN. In this case, you'll have to assign the camera's IP manually. This can be done over your network provider's UI.
```

Comment:
Maybe "network provider's UI" is not exactly descriptive.

Perhaps something like "router's web-based management interface" is better since it is not related to the internet service provider.

Also could be good to explain how to find the management UI IP address.

For linux/mac:
```
netstat -nr | grep default
```

Reply:
Made edit

Reply:
My bad! I think you actually have to use  `ip route | grep default` on Linux

Reply:
Np fixed